### ISSUE TITLE
Multiple Address variables do not work, the first entry is always used

### ISSUE THREAD

**Issue by volth** [gsliepen/tinc#199]
from [the docs](https://www.tinc-vpn.org/documentation/tinc.conf.5)
```
Address = address [port] [recommended]
             The IP address or hostname of this tinc daemon on the real network.  This will only be used when trying to make an
             outgoing connection to this tinc daemon.  Optionally, a port can be specified to use for this address.  Multiple
             Address variables can be specified, in which case each address will be tried until a working connection has been
             established.
```

It seems convenient to specify IPv4 and IPv6 addresses and fall back in case if one of the networks (4 or 6) is down.
But apparently, only the first ```Address``` variable is always used, if meta-connection to the first ```Address``` is timed out or rejected, ```tinc``` gives up and do not try the second ```Address```. Instead it always try to connect to the first address incrementing the timeout between tries. The second ```Address``` is never tried.

**Comment by igel-kun** [comment]
Seconded, please consider merging the patch into the next pre-release, cheers @volth for the effort :)

**Comment by gsliepen** [comment]
Fixed in 1cf2b56.
